File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
import java.util.ArrayList;
class Solution {
    public int[] solution(long n) {
    
    String num = String.valueOf(n);
    int[] answer = new int[num.length()];
    for(int i = num.length()-1 ; i>-1 ; i--){
        answer[num.length()-1-i] =  num.charAt(i) - '0';
    }
    return answer;
    }
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.js
function solution(n) {
    const answer = [];
    const s = n.toString();
    
    for(let i = 0 ; i < s.length ; i++){
        answer.push(s.charCodeAt(s.length - 1 - i) - 48)
    }
    
    return answer;
}